pkg/nmi/server/redirector_windows.go

```diff
@@ -0,0 +1,141 @@
+// +build windows
+
+package server
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+)
+
+type WindowsRedirector struct {
+	Server *Server
+}
+
+func makeRedirectorInt(server *Server) RedirectorInt {
+	return &WindowsRedirector{Server: server}
+}
+
+var podMap = make(map[types.UID]string)
+
+func (s *WindowsRedirector) RedirectMetadataEndpoint() {
+	exit := make(chan struct{})
+	s.Server.PodClient.Start(exit)
+	klog.V(6).Infof("Pod client started")
+
+	s.ApplyRoutePolicyForExistingPods()
+	go s.Sync()
+}
+
+func (s *WindowsRedirector) Sync() {
+	klog.Info("Sync thread started.")
+
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, syscall.SIGTERM, syscall.SIGINT)
+	s.Server.Initialized = true
+
+	var pod *v1.Pod
+
+	for {
+		select {
+		case <-signalChan:
+			s.DeleteRoutePolicyForExistingPods()
+			break
+		case pod = <-s.Server.PodObjChannel:
+			klog.V(6).Infof("Received event: %s", pod)
+
+			fmt.Printf("Node IP and Node Name:%s %s \n", s.Server.HostIP, s.Server.NodeName)
```

Comment:
>fmt.Printf("Node IP and Node Name:%s %s \n", s.Server.HostIP, s.Server.NodeName) [](start = 3, length = 80)

This project uses `klog` not standard `fmt` for logging. You need to use existing logging approach or remove those Printf's